README.md

```diff
@@ -8,10 +8,20 @@
 
 ![havener](.docs/logo.png?raw=true "Havener logo - four stripes symbolising the rank of a harbourmaster inside a gearwheel")
 
+## Table of Contents
+- [Introduction](#introduction)
+- [How do I get started](#getting-started)
+- [Quick Command Overview](#commands-overview)
+- [Configuration](#config)
+- [Contributing](#contributing)
+- [License](#license)
+
+<a name="introduction"></a>
 ## Introducing Havener
 
 Convenience tool to handle tasks around [Containerized CF](https://www.pivotaltracker.com/n/projects/2192232) workloads on a Kubernetes cluster. It deploys multiple Helm Charts using a configuration file, which is used to add in a tiny amount of glue code that is sometimes needed to make things work. Under to cover, `havener` does the same calls that `kubectl` do, nothing special. That means that at the end you have a Helm Release just like you would have using `helm` alone.
 
+<a name="getting-started"></a>
```

Comment:
I do not think we need this. I did a quick test and GitHub generates anchors automatically, for example: https://github.com/homeport/havener#logs-command based on the fact that there is a head line called "logs command".

Reply:
Yes I know, but I wanted to add this in order to get sure that the links stays updated. If I would edit one of the headlines, the anchor link would change but with the HTML tag, the actual anchor is not altered. However, since one would have to update the Table of Contents in most cases anyway, this would not be a big issue. So I could also delete it... What's your opinion about this?

Reply:
I think the markdown linter does not like HTML tags in markdown, even though it is technically supported: https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md033